flytekit/models/task.py

```diff
@@ -398,6 +413,7 @@ def from_flyte_idl(cls, pb2_object):
             custom=_json_format.MessageToDict(pb2_object.custom) if pb2_object else None,
             container=Container.from_flyte_idl(pb2_object.container) if pb2_object.HasField("container") else None,
             task_type_version=pb2_object.task_type_version,
+            config={k: v for k, v in _six.iteritems(pb2_object.config)},
```

Comment:
Is it safe to do that if config is None? or should we add 
```suggestion
            config={k: v for k, v in _six.iteritems(pb2_object.config)} if pb_obj.config is not None else None,
```